src/content/RepoPage/RepoPage.js

```diff
@@ -1,7 +1,73 @@
 import React from 'react';
+import RepoTable from './RepoTable';
+
+const headers = [
+  {
+    key: 'name',
+    header: 'Name',
+  },
+  {
+    key: 'createdAt',
+    header: 'Created',
+  },
+  {
+    key: 'updatedAt',
+    header: 'Updated',
+  },
+  {
+    key: 'issueCount',
+    header: 'Open Issues',
+  },
+  {
+    key: 'stars',
+    header: 'Stars',
+  },
+  {
+    key: 'links',
+    header: 'Links',
+  },
+];
+
+const rows = [
+  {
+    id: '1',
+    name: 'Repo 1',
+    createdAt: 'Date',
+    updatedAt: 'Date',
+    issueCount: '123',
+    stars: '456',
+    links: 'Links',
+  },
+  {
+    id: '2',
+    name: 'Repo 2',
+    createdAt: 'Date',
+    updatedAt: 'Date',
+    issueCount: '123',
+    stars: '456',
+    links: 'Links',
+  },
+  {
+    id: '3',
+    name: 'Repo 3',
+    createdAt: 'Date',
+    updatedAt: 'Date',
+    issueCount: '123',
+    stars: '456',
+    links: 'Links',
+  },
+];
 
 const RepoPage = () => {
-  return <div>REPO PAGE</div>;
+  return (
+    <div className="bx--grid bx--grid--full-width bx--grid--no-gutter repo-page">
+      <div className="bx--row repo-page__r1">
+        <div className="bx--col-lg-16">
+          <RepoTable headers={headers} rows={rows} />
```

Comment:
You're missing the `<div className="bx--col-lg-16">` wrapper here.

Reply:
@kellychurchill Thanks for spotting that somethings wasn’t right. Looks like the problem was a typo in `grid-columns-16` [here](https://github.com/carbon-design-system/carbon-tutorial/pull/1574/commits/e9767a01a793c334d850f151a6b501f3f0f12e7f#diff-5ab26e17c78a72ee4bc1774adbcf305eR3)

I’ve fixed that and looks like styling is right now.